frontend/src/components/markdown/MermaidDiagram.tsx

```
@@ -0,0 +1,89 @@
+          <summary>Show diagram code</summary>
+          <pre>{chart}</pre>
+        </details>
+      </div>
+    );
+  }
+
+  return (
+    <div
+      className="mermaid-wrapper"
+      ref={ref}
+      dangerouslySetInnerHTML={{ __html: svg }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)